src/Windowing/Silk.NET.GLFW/Glfw.cs

```diff
@@ -170,7 +170,7 @@ public static void ThrowExceptions()
         /// Possible errors include <see cref="ErrorCode.InvalidEnum" /> and <see cref="ErrorCode.InvalidValue" />.
         /// </para>
         /// </remarks>
-        public partial void InitHint(InitHint hint, bool value);
+        public partial void InitHint(InitHint hint, int value);
```

Comment:
We are trying to avoid breaking changes in the 2.x series, can you make this an overload instead?